packages/govuk-frontend-review/src/app.mjs

```diff
@@ -133,8 +133,12 @@ export default async () => {
 
     let bodyClasses = ''
 
+    if (exampleConfig.invertLayoutColours) {
+      bodyClasses = `${bodyClasses} app-inverse-example-background`
```

Comment:
Feels like we've now got three battling ways of setting the background colour?

Added to **`<body>`** is `.govuk-template__body` and `.app-inverse-example-background`
Added to **`<html>`** is `.app-no-canvas-background` which already [sets a white background](https://github.com/alphagov/govuk-frontend/commit/8a30d6eaed4215c88525902a0ef0ed3ad384699f)

Can we find a way to unify all our `.app` classes maybe with the `--inverse` modifier?

```css
.app-template {}
.app-template__body {}
.app-template__body--inverse {}
```